java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/build/preprocessor/AbstractPreprocessor.java

```diff
@@ -177,6 +186,8 @@ private String getContainerFile() {
             ENV PROXY_URL=$PROXY_URL
             COPY .jbs/run-build.sh /var/workdir
             COPY . /var/workdir/workspace/source/
+            COPY /mnt/trusted-ca/ca-bundle.crt /etc/pki/ca-trust/source/anchors
+            RUN update-ca-trust
```

Comment:
Is this handled by buildah-oci-ta now ? i.e. we don't need this? 

Question - the `update-ca-trust` is run in buildah-oci-ta - is it also needed in AbstractPreprocessor?

Reply:
Yeah, since AbstractPreprocessor is in the builder image, not the image where buildah-oci-ta runs.